tests/provisioning-manifest-snapshot.yml

```diff
@@ -1 +1,5 @@
 - include: mvn:org.jahia.packages/jahia-additional-modules/JAHIA_VERSION/yaml/provisioning
```

Comment:
The integration tests are failing and it seems to be for the same reason as in https://github.com/Jahia/robots/blob/bc6694281c5186cfde17bcf30fa568ea30ce1086/tests/provisioning-manifest-snapshot.yml#L18
We need to specifically incidate the version of digitall we want to use as the one packaged in 8.1.0.0-SNAPSHOT (and retrieved with mvn:org.jahia.packages/jahia-additional-modules) is still in staging

Reply:
We are not using digital for tests in sitemap. What we are now missing is a templates set so it can't create a site. I guess for the same reasons we couldn't install digitall. Anyways, I'll commit your suggestion, it should install the needed templates set.